lib/Controller/SignalingController.php

```diff
@@ -150,6 +150,13 @@ public function pullMessages($token) {
 		$seconds = 30;
 		$sessionId = '';
 
+		try {
+			$sessionId = $this->session->getSessionForRoom($token);
+			$room = $this->manager->getRoomForSession($this->userId, $sessionId);
+			$room->ping($this->userId, $sessionId, time());
+		} catch (RoomNotFoundException $e) {
```

Comment:
Shouldn't this return a `STATUS_NOT_FOUND`?

Reply:
done